fdbserver/Ratekeeper.actor.cpp

```diff
@@ -646,6 +653,13 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 		if (s.value.valid)
 			totalDiskUsageBytes += s.value.lastReply.storageBytes.used;
 
+	if (now() - self->lastSSListFetchedTimestamp > SERVER_KNOBS->STORAGE_SERVER_LIST_FETCH_TIMEOUT) {
+		limits->tpsLimit = 1;
```

Comment:
If the `tpsLimit` happens to be less than 1, we probably shouldn't increase it or change the reason, even if we can't fetch the storage server list.

Also, is there a particular reason 1 was chosen instead of 0 as the limit?

Reply:
No particular reason. Just a small value. Should i change it to 0?

Reply:
My gut says there isn't a lot to be gained by allowing 1 vs 0, but that the slow trickle of transactions could in some cases cause additional damage. I'd be inclined to clamp it all the way down in this case and only allow immediate priority transactions. You could also potentially check with @etschannen for another opinion.

Reply:
I agree that it should be set to 0